collections-lists/HowToCompareTwoListsThroughOneProperty/HowToCompareTwoListsThroughOneProperty/Methods/Methods.cs

```diff
@@ -0,0 +1,56 @@
+﻿using HowToCompareTwoListsThroughOneProperty.Models;
+
+namespace HowToCompareTwoListsThroughOneProperty.Methods;
+
+public class Methods
```

Comment:
This whole class could be `static`. Also, let's give it a more meaningful name like `ListCompareMethods`

Reply:
Changed to static and renamed to 'ListCompareMethods'.